### PR TITLE
Add extra troubleshooting steps.

### PR DESCRIPTION
**Pull request by markmandel**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup

/kind documentation

> /kind feature
> /kind hotfix

**What this PR does / Why we need it**:

Consolidating more of the troubleshooting steps I usually advocate for
in Slack into our troubleshooting guide, to make helping users with
issues scale better.

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:

Also fixed a small formatting issue in the GameServer reference that we
point to in the troubleshooting doc.

### CODE REVIEW

**Comment by markmandel** [comment]
```
time="2020-07-28 21:56:06.424" level=info msg="error waiting for fleet condition" error="timed out waiting for the condition" fleet=simple-fleet-cqt7j
time="2020-07-28 21:56:06.698" level=info msg="Namespace gsa-multicluster-local-596fe0ac-cbaa-4e6d-b505-95886f301649 is deleted"
--- FAIL: TestMultiClusterAllocationOnLocalCluster (0.01s)
    --- FAIL: TestMultiClusterAllocationOnLocalCluster/Distributed (323.80s)
        gameserverallocation_test.go:98: error waiting for fleet condition on fleet simple-fleet-cqt7j
FAIL
time="2020-07-28 21:56:06.890" level=info msg="Namespace 1595972959 is deleted"
FAIL	agones.dev/agones/test/e2e	407.170s
FAIL
```

Weird

File: site/content/en/docs/Guides/troubleshooting.md [hunk 1]
```diff
@@ -9,6 +9,38 @@ description: "Troubleshooting guides and steps."
 
 If there is something going wrong with your GameServer, there are a few approaches to determining the cause:
 
+### Run with the local sdk server
+
+A good first step for seeing what may be going wrong, is replicating the issue locally. To do this you can take
```
Comment (aLekSer) [inline site/content/en/docs/Guides/troubleshooting.md:14]:
```suggestion
A good first step for seeing what may be going wrong is replicating the issue locally. To do this you can take
```

File: site/content/en/docs/Guides/troubleshooting.md [hunk 2]
```diff
@@ -9,6 +9,38 @@ description: "Troubleshooting guides and steps."
...
+advantage for the Agones [local sdk server]({{% ref "/docs/Guides/local-game-server.md" %}})
```
Comment (aLekSer) [inline site/content/en/docs/Guides/troubleshooting.md:15]:
```suggestion
advantage of the Agones [local SDK server]({{% ref "/docs/Guides/local-game-server.md" %}})
```

File: site/content/en/docs/Guides/troubleshooting.md [hunk 3]
```diff
@@ -9,6 +9,38 @@ description: "Troubleshooting guides and steps."
...
+, with the following troubleshooting steps:
+
+1. Run your game server as a local binary against the local sdk server
```
Comment (aLekSer) [inline site/content/en/docs/Guides/troubleshooting.md:18]:
```suggestion
1. Run your game server as a local binary against the local SDK server
```

File: site/content/en/docs/Guides/troubleshooting.md [hunk 4]
```diff
@@ -9,6 +9,38 @@ description: "Troubleshooting guides and steps."
...
+1. Run your game server container against the local sdk server. It's worth noting that running with
```
Comment (aLekSer) [inline site/content/en/docs/Guides/troubleshooting.md:19]:
```suggestion
1. Run your game server container against the local SDK server. It's worth noting that running with 
```

File: site/content/en/docs/Guides/troubleshooting.md [hunk 5]
```diff
@@ -9,6 +9,38 @@ description: "Troubleshooting guides and steps."
...
 
+### Run with the local sdk server
+
+A good first step for seeing what may be going wrong, is replicating the issue locally. To do this you can take
+advantage for the Agones [local sdk server]({{% ref "/docs/Guides/local-game-server.md" %}})
+, with the following troubleshooting steps:
+
+1. Run your game server as a local binary against the local sdk server
+1. Run your game server container against the local sdk server. It's worth noting that running with 
+   `docker run --network=host ...` can be an easy way to allow your game server container(s) access to the local sdk
```
Comment (aLekSer) [inline site/content/en/docs/Guides/troubleshooting.md:20]:
```suggestion
   `docker run --network=host ...` can be an easy way to allow your game server container(s) access to the local SDK
```

File: site/content/en/docs/Guides/troubleshooting.md [hunk 6]
```diff
@@ -9,6 +9,38 @@ description: "Troubleshooting guides and steps."
...
+    server)  
+
+At each stage, keep an eye on the logs of your game binary, and the local sdk server, and ensure there are no system
```
Comment (aLekSer) [inline site/content/en/docs/Guides/troubleshooting.md:23]:
```suggestion
At each stage, keep an eye on the logs of your game server binary, and the local SDK server, and ensure there are no system
```

File: site/content/en/docs/Guides/troubleshooting.md [hunk 7]
```diff
@@ -9,6 +9,38 @@ description: "Troubleshooting guides and steps."
 
 If there is something going wrong with your GameServer, there are a few approaches to determining the cause:
 
+### Run with the local sdk server
```
Comment (aLekSer) [inline site/content/en/docs/Guides/troubleshooting.md:12]: Just one more left 
```suggestion
### Run with the local SDK server
```

**Pull request merged**